tested/main.py

```diff
@@ -3,37 +3,53 @@
 """
 
 import os
+from pathlib import Path
 from typing import IO
 
 from tested.configs import DodonaConfig, create_bundle
 from tested.dsl import parse_dsl
+from tested.nat_translation import run_translation
 from tested.testsuite import parse_test_suite
 
 
-def run(config: DodonaConfig, judge_output: IO):
+def run(config: DodonaConfig, judge_output: IO, language: str | None = None):
     """
     Run the TESTed judge.
 
     :param config: The configuration, as received from Dodona.
     :param judge_output: Where the judge output will be written to.
+    :param language: The language to use to translate the test-suite.
     """
-    try:
```

Comment:
if you edit `run_translation` to be able to take a string as input, you could nicely keep doing this read in all cases

then you could do 
```
if language: 
   textual_suite, missing_keys = run_translation(textual_suite, language)
   ```